### ISSUE TITLE
[BUG] Crash after boot

### ISSUE THREAD

**Issue by opk12** [FriesI23/mhabit#142]
I upgraded 1.10.2 to 1.10.5 from F-droid, on Android 7.1. Now whenever I reboot, the first time I open the app, the system popup shows a crash that `Unable to start receiver [...]ScheduledNotificationBootReceiver`. The notifications setting is disabled, but I would like to try it in the future.

The app works normally otherwise.

```
FATAL EXCEPTION: main
Process: io.github.friesi23.mhabit, PID: 1621
java.lang.RuntimeException: Unable to start receiver com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver: java.lang.RuntimeException: Missing type parameter.
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3398)
	at android.app.ActivityThread.-wrap20(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1743)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6816)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)
Caused by: java.lang.RuntimeException: Missing type parameter.
	at x.a.d(SourceFile:1)
	at x.a.<init>(SourceFile:1)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin$a.<init>(SourceFile)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.loadScheduledNotifications(SourceFile)
	at com.dexterous.flutterlocalnotifications.FlutterLocalNotificationsPlugin.rescheduleNotifications(SourceFile)
	at com.dexterous.flutterlocalnotifications.ScheduledNotificationBootReceiver.onReceive(SourceFile)
	at android.app.ActivityThread.handleReceiver(ActivityThread.java:3391)
	... 8 more
```

**Comment by Miaaaim** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[BUG] Crash after boot](https://trello.com/c/Ve8iM7lz/68-bug-crash-after-boot)

**Comment by FriesI23** [comment]
it's seen like a compatibility issue with flutter_local_notification and AGP8, fixed #144

Thank you for your feedback 😉.
